src/lib/draw/polygon/polygon_tool.ts

```diff
@@ -82,6 +84,8 @@ export class PolygonTool {
   finish() {
     let polygon = this.polygonFeature();
     if (polygon) {
+      // TODO RouteTool passes a copy to each callback for paranoia. Should we
```

Comment:
Unrelated to this PR, just something I noticed while touching this code again. There were bugs long ago with split route mode, and as a response, I made defensive copies to callbacks. Might be good for us to do the same everywhere for consistency, since we haven't got anything like Rust's borrow checker to prevent us from accidentally modifying something passed to a callback.